packages/passport/sdk-sample-app/src/context/ImmutableProvider.tsx

```diff
@@ -41,6 +41,7 @@ const getCoreSdkConfig = (environment: EnvironmentNames) => {
 
 const getPassportConfig = (environment: EnvironmentNames): PassportModuleConfiguration => {
   const sharedConfigurationValues = {
+    headers: { 'x-sdk-version': 'ts-immutable-sdk-1.0.0' },
```

Comment:
Hey @scastleman-immutable 
Nik also mentioned the **x-sdk-version** is not overridable.
so I think that's probably not going to work

Reply:
Here is the test to ensure it not override
https://github.com/immutable/ts-immutable-sdk/blob/main/packages/internal/generated-clients/src/config/config.test.ts#L24

Reply:
They weren't overridable, but I made changes so this gets passed through. Everything worked in testing on my end, but have a go yourself and see if they work for you too

Reply:
@CodeSchwert, can you confirm that the header is overriderable?
I did a test myself and noticed that it is overridden. But as you said to me earlier, We didn't expect **createConfig** [function ](https://github.com/immutable/ts-immutable-sdk/blob/main/packages/internal/generated-clients/src/config/config.ts#L21) overridable, 
so my understanding is even if we change the passport config to allow the config override. it still couldn't override the header. I was confused there.